lib/runcible/extensions/repository.rb

```diff
@@ -31,20 +31,18 @@ def create_with_distributors(id, distributors)
       # @param  [Hash]                                  optional      container for all optional parameters
       # @return [RestClient::Response]                                the created repository
       def create_with_importer_and_distributors(id, importer, distributors = [], optional = {})
-        if importer.is_a?(Runcible::Models::Importer)
+        if importer && importer.is_a?(Runcible::Models::Importer)
           optional[:importer_type_id] = importer.id
           optional[:importer_config] = importer.config
-        else
+        elsif importer
           optional[:importer_type_id] = importer.delete('id') || importer.delete(:id)
           optional[:importer_config] = importer
-        end if importer
+        end
 
         repo_type = if importer.methods.include?(:repo_type)
                       importer.repo_type
                     elsif importer.is_a?(Hash) && importer.key?(:repo_type)
                       importer[:repo_type]
-                    else
-                      nil
```

Comment:
I like the `Style::EmptyElse` cop but I feel like it should ignore assignments like this. This technically works without the `else` though.